collectionFunctions.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/** Displays each pokemon and stats
+ * @param array $pokemon
+ * @return String
+ */
+function displayPoke(Array $pokemon) : String {
```

Comment:
Picky but PSR-12 likes type-hinting to be a certain way (had this flagged by Ash myself): colon next to bracket 

https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-returntypesspacing-php

Reply:
Thanks! Have updated now